ansible/run.yml

```diff
@@ -0,0 +1,9 @@
+---
+- hosts: local
+  connection: local
+  vars_files:
+      - runtime_vars.yml
+  roles:
+      - facts
+      - nodejs
```

Comment:
@avtar after what we talked about last night, the best solution I came up with (which I don't love) is to `pkill supervisord` after running the tests, then start it again as the final step of the container's entrypoint. This:

- lets the test playbook run
- fails the container launch if the test playbook doesn't complete successfully
- launches the container application in the foreground once the testing is complete

I don't love this but I think it will do for the moment.